drivers/bridge/bridge.go

```diff
@@ -46,6 +42,8 @@ type Configuration struct {
 	EnableICC             bool
 	EnableIPForwarding    bool
 	AllowNonDefaultBridge bool
+	// Driver components
+	PortAllocator *portallocator.PortAllocator
```

Comment:
@mrjana I think we both are in sync for both points 1 & 2 :-)
>> if we dont make it a singleton for that particular namespace, then we are essentially not doing the right thing
In my comment on 2, my point was to keep the singleton per namespace and not a package level singleton.